rust/hyperlane-base/src/settings/mod.rs

```diff
@@ -184,6 +184,19 @@ impl Settings {
         Ok(result)
     }
 
+    /// Try to get a map of chain name -> validator announce contract
+    pub async fn build_all_validator_announces(
```

Comment:
is this used anywhere?

makes me realize there's no reason for the relayer to be using `build_all_interchain_gas_paymasters`...

Reply:
Good catch, cleaned this up